lib/guests.js

```diff
@@ -8,40 +8,28 @@ function guests(bot, message) {
   debug('begin');
   let user = message.user;
   let getData = Promise.promisify(bot.botkit.storage.users.get);
+  let guestsInviteResults = {
+    ok: 'Invitado',
+    already_invited: 'Ya se invitó',
+    already_in_team: 'Ya está registrado',
+  };
 
   return getData(user)
     .then((data) => {
       debug('pana', data);
       debug('panas', data.guests);
 
       if (data.guests.length === 0) {
-        return bot.reply(message, 'Oiga, invitá un parcero primero!');
-      }
-
-      const activeGuests = data.guests.filter((guest) => guest.result === 'ok');
-      if (activeGuests.length === 0) {
-        return bot.reply(message, 'No hay parceros activos, invitá uno ome!');
+        return bot.reply(message, 'Oe, invitá un parcero primero!');
       }
 
       return bot.reply(message, [
           'Tus parceros:',
           '\n',
-          `${activeGuests.map((guest) => guest.guest).join('\n')}`,
+          `${data.guests.map((guest) => `${guest.guest}: ${guestsInviteResults[guest.result]}`).join('\n')}`,
```

Comment:
It would look like this:
![screen shot 2016-09-06 at 11 30 49](https://cloud.githubusercontent.com/assets/1002461/18282148/6bc8c962-7425-11e6-8d75-5aa50e1a76bb.png)